### ISSUE TITLE
[Question] What makes it impossible for the fan speed to go below 4%?

### ISSUE THREAD

**Issue by RicArch97** [mcgillij/amdfan#8]
I found this to be a bit annoying, since as you know, on AMD Adrenaline in Windows you can enable `Zero RPM` where the fans don't spin when the card temperature is below 50C. This is also the case with the default `auto` settings of the card.

**Comment by mcgillij** [comment]
There was/is a driver bug on some cards: https://gitlab.freedesktop.org/drm/amd/-/issues/1164
I only have a Vega64 and a 6800xt to test with and I haven't been able to reproduce it on those.

You could comment out the 2 lines here: https://github.com/mcgillij/amdfan/blob/089f987c3cedb9192e96856ab24537b0b4eb2592/src/amdfan/amdfan.py#L284 if you wanted to try to reproduce it.

**Comment by mazunki** [comment]
Just did some measurements myself, seems like it depends on the hardware used. Maybe we should change it from a `ValueError` to a `LOGGER.debug`?

![image](https://github.com/user-attachments/assets/b7da13d9-4ed8-4988-b11b-9ea96a768efd)

**Comment by mcgillij** [comment]
Yeah it was a hardware limitation at the time, it's been several years since I've had to work around it. So it could be partially resolved at this point.
